hail/python/hail/expr/functions.py

```diff
@@ -48,6 +49,17 @@ def _quantile_from_cdf(cdf, q):
     return cdf.values[idx]
 
 
+@typecheck(cdf=expr_struct(), confidence=expr_oneof(expr_float32, expr_float64))
+def _error_from_cdf(cdf, confidence):
+    """Estimates error of approx_cdf aggregator, using Hoeffding's inequality.
+
+    Error of any one quantile estimation is less than returned bound with
+    probability 1-'confidence'.
+    """
+    s = range(0, len(cdf._compaction_counts)).map(lambda i: cdf._compaction_counts[i] << (2*i)).sum
```

Comment:
does the `.sum` notation actually work?

Reply:
Also, I know most of these functions exist directly in the namespace, but I think it's nice to prefix with `hl.` anyways to make it clear that we're operating on expressions. We also may want to bind the `cdf` expression so we're not potentially computing the aggregation twice, although if it's not bound outside the scope of this function then we'd need another computation to get the actual cdf anyways?